mundane-io/src/main/scala/com/ambiata/mundane/io/Archive.scala

```diff
@@ -9,18 +10,18 @@ import scala.sys.process._
 import scalaz._, Scalaz._, \&/._
 import scalaz.effect._
 
-case class Archive(archiveFile: FilePath, checksumFile: FilePath, contents: List[FilePath], checksum: Checksum) {
-  def files: List[FilePath] =
+case class Archive(archiveFile: LocalFile, checksumFile: LocalFile, contents: List[LocalFile], checksum: Checksum) {
+  def files: List[LocalFile] =
     List(archiveFile, checksumFile)
 }
 
 object Archive {
```

Comment:
Consider removing this, it would be nice to have something like this, but it isn't used anywhere any more and It isn't really up to scratch.